src/mediaengine/open_cdm_mediaengine_factory.h

```diff
@@ -21,12 +21,12 @@
 
 #include "media/cdm/ppapi/external_open_cdm/src/mediaengine/open_cdm_mediaengine.h"
 #include "media/cdm/ppapi/external_open_cdm/src/mediaengine/open_cdm_mediaengine_impl.h"
-
+#include "media/cdm/ppapi/external_open_cdm/src/browser/opencdm_features.h"
 #include "media/cdm/ppapi/cdm_logging.h"
```

Comment:
These include statements are Chromium specific. There will be a need to rebase the 'chromium-65' branch on the 'chromium-53' branch. See the following commit where they were modified to support WPE:

git show 109ca12dcb772fa05a663d165ae7be2d43f442b7